maro/simulator/scenarios/oncall_routing/duration_time_predictor.py

```diff
@@ -16,15 +17,21 @@ class TimePredictionFeature:
 
 
 class EstimatedDurationPredictor:
+    def __init__(self) -> None:
+        self._cache: Dict[Tuple[Coordinate, Coordinate], int] = {}
+
     def predict(
         self,
         tick: int,
         source_coordinate: Coordinate,
         target_coordinate: Coordinate,
         feature: TimePredictionFeature = None
-    ) -> float:
-        distance = geo_distance_meter(source_coordinate, target_coordinate)
-        return math.ceil(max(1.0, distance / 200.0))  # TODO: fake
+    ) -> int:
+        key = (source_coordinate, target_coordinate)
```

Comment:
For this estimated one, the key could be (min(src, dest), max(src, dest))?

Reply:
For now it is okay to do so. I'll change this.